src/models/components.d.ts

```diff
@@ -0,0 +1,64 @@
+import { SvgIconComponent } from "@mui/icons-material";
+
+/*=============================================
+=            // 1,  factSearchLists            =
+=============================================*/
+
+export interface listItemTypes {
+  name: string;
+  path: string;
+  image: string;
+}
+export interface GroupListsTypes {
```

Comment:
You are declaring one type here, but in name you are using plural. This is not correct. This name should be `GroupedListType` instead of `GroupListTypes`. Also change the name according to updated component name.

Reply:
done